types/tokenizr/index.d.ts

```diff
@@ -0,0 +1,264 @@
+// Type definitions for tokenizr 1.5
+// Project: https://github.com/rse/tokenizr
+// Definitions by: Nicholas Sorokin <https://github.com/aNickzz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export {};
+
+export default class Tokenizr {
```

Comment:
It's not `export default`: https://github.com/rse/tokenizr/blob/master/src/tokenizr.js#L713

Reply:
I've removed "default" however I'm not sure of the correct way to declare the export. With the "export default" I was able to build with these types in another project (I'll test this again tonight with the changes I've just comitted)